site/public/js/admin-gradeable-updates.js

```diff
@@ -241,7 +278,59 @@ function ajaxCheckBuildStatus() {
         }
     });
 }
-
+function setRandomGraders(gradeable_id,p_values,successCallback,errorCallback)
```

Comment:
Please fix/add the indentation throughout this file for if statements/loops, as you have done above